tools/fv_diagnostics.F90

```diff
@@ -143,14 +196,16 @@ subroutine fv_diag_init(Atm, axes, Time, npx, npy, npz, p_ref)
     rainwat = get_tracer_index (MODEL_ATMOS, 'rainwat')
     snowwat = get_tracer_index (MODEL_ATMOS, 'snowwat')
     graupel = get_tracer_index (MODEL_ATMOS, 'graupel')
+    o3mr    = get_tracer_index (MODEL_ATMOS, 'o3mr')
+    cld_amt = get_tracer_index (MODEL_ATMOS, 'cld_amt')
 
 ! valid range for some fields
 
 !!!  This will need mods for more than 1 tile per pe  !!!
 
     vsrange = (/ -200.,  200. /)  ! surface (lowest layer) winds
 
-    vrange = (/ -300.,  300. /)  ! winds
```

Comment:
Not sure if this is relevant, but i think vrange = (/ -300.,  300. /) in xanadu

Reply:
Thanks for pointing that out.  The valid range for horizontal winds has been expanded to (-330,330).  This expansion was done by the science team and does cause metadata differences for diagnostics if horizontal wind fields are present.